style.css

```diff
@@ -0,0 +1,120 @@
+@import url("https://fonts.googleapis.com/css2?family=Kosugi+Maru&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Fira+Sans:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Kosugi+Maru&display=swap");
+
+body {
+  background-color: #eeb3c4;
+  margin: 20px;
+}
+
+header {
+  display: flex;
+  column-gap: 10px;
+  align-items: center;
+  justify-content: center;
+}
+
+h1 {
+  font-family: "Kosugi Maru";
+  color: #000000;
+  font-size: 70px;
+  margin: 50px;
+  display: flex;
+  letter-spacing: 0.1, 5rem;
+  
+}
+
+h2 {
+  font-family: "Fira Sans";
+}
+
+/*hashtag on books because its an ID */
+#books {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-top: 50px;
+  column-gap: 70px;
+  row-gap: 40px;
+}
+
+.book:hover {
+  box-shadow: -1px 0px 11px 1px rgba(138, 31, 222, 0.75);
+}
+
+.book {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 10px;
+  max-width: 200px;
+  font-family: "Fira Sans";
+  background-color: rgb(251, 251, 206);
+  border-radius: 20px;
+  /* Overflow hidden stops the image from popping out from the borders */
+  overflow: hidden;
+  padding: 11px;
+}
+
+.book-image {
+  width: 200px;
+  border-radius: 10px;
+}
+
+.book-title {
+  font-weight: bold;
+  margin-top: 20px;
+  font-family: "Fira Sans";
+}
+
+select {
+  background-color: #e36262;
+  color: white;
+  border: none;
+  border-radius: 30px;
+  padding: 18px 20px;
+  margin-right: 4px;
+  font-size: 16px;
+  line-height: 26px;
+  font-family: "Fira Sans";
+  font-weight: 500;
+  cursor: pointer;
+  transition: all 0.3s ease;
+  margin-top: 4px;
+}
+
+button {
+  background-color: #bc86dd;
+  color: white;
+  border: none;
+  border-radius: 30px;
+  padding: 16px 20px;
+  margin-right: 25px;
+  font-size: 16px;
+  line-height: 26px;
+  font-family: "Fira Sans";
+  font-weight: 500;
+  cursor: pointer;
+  transition: all 0.3s ease;
```

Comment:
When you apply an effect to the button, like in this case the transition, is making all the UI around move, and this is to be avoided. A solution would be to wrap the button in a div and give the div a fixed height 👍

Reply:
Thanks! It's fixed!